src/RTC/eventtarget.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const process = require('process');
```

Comment:
Can it be?

```
const {process} = global;
```

Would save a function call.

Reply:
1739737

Reply:
that worked for me.